src/Math-Matrix/PMMatrix.class.st

```diff
@@ -311,6 +311,12 @@ PMMatrix >> atColumn: aColumnNumber put: aCollection startingAt: rowNumber [
 
 ]
 
+{ #category : #'cell accessing' }
+PMMatrix >> atRow: anInteger [
```

Comment:
`anInteger` -> `rowNumber` . Here should be 0 index it, do you think?

Reply:
By cell accessing, you mean accessing a matrix element? If so, maybe that's a more domain-driven naming. Again, thoughts welcome.

Reply:
> `anInteger` -> `rowNumber` . Here should be 0 index it, do you think?

Yes, `rowNumber` makes more sense. But `atColumn:` method had the argument name as `anInteger`, so I thought it would be better to stick on to the same here also. Shall I change it to `rowNumber`?

Reply:
> By cell accessing, you mean accessing a matrix element? If so, maybe that's a more domain-driven naming. Again, thoughts welcome.

Yeah, cell accessing doesn't seem the exact category for this method but again I stuck on to this because `atColumn:` method also had the same category.

Reply:
> > `anInteger` -> `rowNumber` . Here should be 0 index it, do you think?
> 
> Yes, `rowNumber` makes more sense. But `atColumn:` method had the argument name as `anInteger`, so I thought it would be better to stick on to the same here also. Shall I change it to `rowNumber`?

Yes please. It makes the code clearer, I think. 

I would suggest you do the same for the argument name for atColumn. Great spot

Reply:
> > By cell accessing, you mean accessing a matrix element? If so, maybe that's a more domain-driven naming. Again, thoughts welcome.
> 
> Yeah, cell accessing doesn't seem the exact category for this method but again I stuck on to this because `atColumn:` method also had the same category.

Leave it as it is in this PR, and propose a separate one for improving the category name.